topics/teaching/metadata.yaml

```diff
@@ -12,9 +12,18 @@ summary: |
 
     The Galaxy community via the Galaxy Training Network supports instructors and
     workshop organizers using with a series of recommandations (tutorials below) and
-    regular discussion meetings.<br>
+    regular discussion meetings.
 
-    You can additionally <a href="philosophies/">read about our training philosophies.</a>
+subtopics:
+  - id: pedagogy
+    title: "Pedagogical aspects"
+    description: "These will guide you on the pedagogy to deliver impactfully trainings"
```

Comment:
I think the subtopic heading will still show up on the topic page, even though all the tutorials for this subtopic are disabled, maybe add `(work in progress)` to the end of the sentence so it doesn't look weird? (tho I think you could also just enable them now, and make updates after your pilot workshop, but that's up to you)

Reply:
I was also thinking about enabling the 5 main tutorials directly

Reply:
go for it!

Reply:
I did it yesterday 🙂